genome_designer/genome_finish/millstone_de_novo_fns.py

```diff
@@ -1,21 +1,60 @@
 import subprocess
 import os
 import shutil
+import pickle
 import re
 
 from settings import TOOLS_DIR
 from settings import SAMTOOLS_BINARY
 from settings import BASH_PATH
+from genome_finish import __path__ as gf_path_list
 
-def get_unmapped_reads(bam_filename, output_filename):
+GENOME_FINISH_PATH = gf_path_list[0]
+VELVETH_BINARY = TOOLS_DIR + '/velvet/velveth'
+VELVETG_BINARY = TOOLS_DIR + '/velvet/velvetg'
 
-    unmapped_reads_bam_file = output_filename
 
-    cmd = '{samtools} view -h -b -f 0x4 {bam_filename}'.format(
+def get_clipped_reads(bam_filename, output_filename):
+    cmd = ' | '.join([
+        '{samtools} view -h {bam_filename}',
+        '{extract_clipped_script} -i stdin',
+        '{samtools} view -Sb -']).format(
             samtools=SAMTOOLS_BINARY,
-            bam_filename=bam_filename)
+            bam_filename=bam_filename,
```

Comment:
generally use two indents in these situations. And several similar situations in this commit.